Dockerfile.fuzz

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as builder
+
+RUN apk add --no-cache python3 py3-pip bash coreutils
+
+RUN wget https://github.com/microsoft/restler-fuzzer/archive/refs/tags/v9.2.2.tar.gz
```

Comment:
Sonarqube is flagging this line:

https://sonarcloud.io/project/security_hotspots?pullRequest=4569&id=edgexfoundry_edgex-go&tab=how_to_fix